### PR TITLE
Fixes #23225: 

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/23225

### CODE REVIEW

File: webapp/sources/rudder/rudder-rest/src/test/scala/com/normation/rudder/rest/SharedFilesApiTest.scala [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/*
+ *************************************************************************************
+ * Copyright 2018 Normation SAS
```
Comment (fanf) [inline webapp/sources/rudder/rudder-rest/src/test/scala/com/normation/rudder/rest/SharedFilesApiTest.scala:3]: bad year

**Review by fanf — APPROVED** [review]
LGTM

**Comment by amousset** [comment]
OK, squash merging this PR

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**